packages/@react-spectrum/list/package.json

```diff
@@ -33,6 +33,7 @@
   "dependencies": {
     "@babel/runtime": "^7.6.2",
     "@react-aria/button": "^3.6.1",
+    "@react-aria/dnd": "3.0.0-alpha.12",
```

Comment:
?

Reply:
argh, IDE auto added probably, thanks for catching